examples/jsm/helpers/OctreeHelper.js

```diff
@@ -39,17 +53,20 @@ class OctreeHelper extends LineSegments {
 
 		}
 
-		traverse( octree.subTrees );
+		traverse( this.octree.subTrees );
 
-		const geometry = new BufferGeometry();
-		geometry.setAttribute( 'position', new Float32BufferAttribute( vertices, 3 ) );
+		if ( this.geometry.attributes.position && this.geometry.attributes.position.count === vertices.length ) {
```

Comment:
If would simplify this and remove this check. When an Octree changes, it's usually a structural change ending in a different number of vertices.

Reply:
done.